internal/api/http/dataplane/client.gen.go

```diff
@@ -0,0 +1,236 @@
+// Package dataplane provides primitives to interact with the openapi HTTP API.
```

Comment:
We've now introduced a dataplane client for our own internal use (it's private to this project). Are we sure we don't want to export this for external use?

Reply:
Moving to root directory of project